src/trashinfo.rs

```diff
@@ -14,10 +14,21 @@ impl Trashinfo {
             2: deletion_date.to_string(),
         }
     }
+    
     pub fn to_contents(&self) -> String {
         let pct_string = percent_encode(&self.1);
         format!("{}\nPath={}\nDeletionDate={}\n", self.0, pct_string, self.2)
     }
+
+    pub fn from_file(path: &str) -> io::Result<Trashinfo> {
+        let mut lines = read_lines(path)?.skip(1);
+        let mut path_and_filename = lines.next().expect("Failed to read the path in trashinfo.")?;
+        path_and_filename.replace_range(..5, "");
```

Comment:
To make things more robust, I think there should be a condition here to make sure that the first 5 chars of the string match "Path=".